packages/@ember/-internals/glimmer/lib/utils/string.ts

```diff
@@ -2,15 +2,17 @@
 @module @ember/template
 */
 
-export class SafeString {
-  public string: string;
+import type { SafeString as GlimmerSafeString } from '@glimmer/runtime';
+
+export class SafeString implements GlimmerSafeString {
```

Comment:
I realized after landing this: we'll need docs here to make it show up appropriately. I'll follow up to that effect tomorrow.